DCO.md

```diff
@@ -0,0 +1,44 @@
+# JerryScript Developer's Certificate of Origin
+
+The JerryScript project uses the signed-off-by language and process, to give us a clear chain of trust for every patch received.
+
+> By making a contribution to this project, I certify that:
+
+> (a)	The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or
+
+> (b)	The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or
+
+> (c)	The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.
+
+> (d)	I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project, under the same open source license.
+
+
+### Using the Signed-Off-By Process
+
+We have the same requirements for using the signed-off-by process as the Linux kernel. In short, you need to include a signed-off-by tag in every patch:
+
+"Signed-off-by:" this is a developer's certification that he or she has the right to submit the patch for inclusion into the project. It is an agreement to the Developer's Certificate of Origin (above). **Code without a proper signoff cannot be merged into the mainline.**
```

Comment:
The beginning of the paragraph is misleading. It's not "Signed-off-by:" but "JerryScript-DCO-1.0-Signed-off-by:", as stated below as well. This might seem picky, but we have seen problems from such slight misinterpretations...